csm_web/scheduler/factories.py

```diff
@@ -45,7 +45,7 @@ class Meta:
 
 
 BUILDINGS = ("Cory", "Soda", "Kresge", "Moffitt")
-DAY_OF_WEEK_DB_CHOICES = [db_value for db_value, display_name in Spacetime.DAY_OF_WEEK_CHOICES]
+DAY_OF_WEEK_DB_CHOICES = [db_value for db_value, display_name in Spacetime.DayOfWeek.choices]
```

Comment:
We don't need this comprehension anymore, `DAY_OF_WEEK_DB_CHOICES` can be replaced with `Spacetime.DayOfWeek.values` everywhere it is used.

https://docs.djangoproject.com/en/3.0/ref/models/fields/#field-choices-enum-types